packages/react/src/contexts/ClerkContextProvider.tsx

```diff
@@ -74,11 +74,11 @@ export function ClerkContextProvider(props: ClerkContextProvider): JSX.Element |
     <IsomorphicClerkContext.Provider value={clerkCtx}>
       <ClientContext.Provider value={clientCtx}>
         <SessionContext.Provider value={sessionCtx}>
-          <OrganizationContext.Provider value={organizationCtx}>
+          <OrganizationProvider {...organizationCtx.value}>
```

Comment:
I guess we're treating this differently because of `swrConfig`... Not sure how I feel about this small discrepancy to be honest - if this is a testing-only issue, we might be able to find a different solution that works for the tests. What was the original issue?

Reply:
We need a way to clear cache between tests. The recommended way from the SWR docs [is this](https://swr.vercel.app/docs/advanced/cache#reset-cache-between-test-cases)

Wrapping our `CoreClerkContextWrapper` with <SWRConfig /> will not work. I verified this by logging the results of useSWRConfig inside our `useOrganization` while tests where running.

My solution was to expose a OrganizationProvider instead of the "low-level" context and include the `<SWRConfig/> provider` inside of the `OrganizationProvider`. 
☝️ **_This is now working as expecting._**

Reply:
I see that the cache provider  follows the following interface: 
```
interface Cache<Data> {
  get(key: string): Data | undefined
  set(key: string, value: Data): void
  delete(key: string): void
  keys(): IterableIterator<string>
}
```

Did you try wrapping the test contexts with a `SWRConfig`  that uses a "noop" cache provider, eg: 
```
{
get: () => undefined,
set: noop,
delete: noop,
keys: () => []
}
```

so that we don't need to reset it between the tests?

Reply:
I don't see why this would help. It seemed like the useSWRConfig within useOrganization would return the default values, as it couldn't reach the correct provider of SWRConfig. 

For example is I was passing <SWRConfig value={{ dedupingInterval: 0 }}/> the useSWRConfig would return `{dedupingInterval: 2000}` which is the default value

Reply:
Does this mean that a SWRConfig provider is found deeper in the tree so you cannot override the defaults in the tests?